R/zzz.R

```diff
@@ -249,8 +249,8 @@ settings <- NULL
   toset <- !(names(op_lintr) %in% names(op))
   if (any(toset)) options(op_lintr[toset])
 
-  # This is just here to quiet R CMD check
-  if (FALSE) backports::import
+  # requires R>=3.6.0
+  backports::import(pkgname, "str2lang")
```

Comment:
Is this not the proper way to use backports? It seems to have failed to import `str2lang` in our older tests...

Reply:
Looks like a {backports} bug maybe? https://github.com/r-lib/backports/issues/68